crates/summit/migrations/20250403180711_init.sql

```diff
@@ -58,17 +57,18 @@ CREATE TABLE IF NOT EXISTS task (
     status TEXT NOT NULL,
     allocated_builder TEXT,
     log_path TEXT,
-    started BIGINT NOT NULL DEFAULT (unixepoch()),
-    updated BIGINT NOT NULL DEFAULT (unixepoch()),
+    added BIGINT NOT NULL DEFAULT (unixepoch ()),
```

Comment:
We should add a new migration as to not brick existing deployments since this migration file would have already been run.

Also, in the new migration after adding `added` & modifying `started` to make it nullable / remove the default, we should set `added` equal to `started` with an UPDATE statement for backwards compatibility.

Reply:
Oops, I meant to ask about this. I had something similar to what you are suggesting but summit always failed when running 'just up' so I had did it this way to get things working while testing. As I write this I think I just realized what it was, being use to MySQL I was using ALTER COLUMN but this being sqlite that does not exist. A quick read seems I need to create a new temp table (with altered columns), copy data from current into temp, drop current, rename temp to proper name.

Reply:
You should be able to do an ALTER TABLE ADD / MODIFY COLUMN.

Then right afterward do an UPDATE statement to copy started to added.

Reply:
Looking here: https://www.sqlite.org/lang_altertable.html I don't see a MODIFY COLUMN. Point 7 seems to show that I would need to do this routine: 

1. Create new table
2. Copy data
3. Drop old table
4. Rename new into old

Or possibly the: PRAGMA/UPDATE schema method.

But if I am missing something please let me know.

Reply:
Shit my bad, I got my wires crossed too xD 

We can then rename started to created and then add started after?